[bitnami/tomcat] TOMCAT_EXTRA_JAVA_OPTS are added multiple times to tomcat-env.sh #53082

christian-scheible opened the pull request:
Preventing `TOMCAT_EXTRA_JAVA_OPTS` to be added multiple times to _tomcat-env.sh_ when the container is restarted.

Signed-off-by: Christian Scheible <christian.scheible@uni-konstanz.de>

### Description of the change

This changes checks if the additional **TOMCAT_EXTRA_JAVA_OPTS** have allready been added to _tomcat-env.sh_ before adding them. Otherwise every restart will add them again to the file.

### Benefits

This prevents adding options multiple times to the **JAVA_OPTS** which fails for example if someone want's to attach a debugger like this: `-agentlib:jdwp=transport=dt_socket,address=*:9543,suspend=n,server=y`

### Possible drawbacks

-

### Applicable issues

#53082

### Additional information

-

andresbono reviewed (commented):
Thank you for your contribution! I think it makes sense. Could you please take a look at my comment?

Commit:
[bitnami/php-fpm] Release 8.1.26-debian-11-r4 (#53325)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql] Release 13.13.0-debian-11-r16 (#53326)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql] Release 12.17.0-debian-11-r16 (#53327)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql-repmgr] Release 12.17.0-debian-11-r10 (#53328)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql-repmgr] Release 16.1.0-debian-11-r10 (#53329)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql] Release 16.1.0-debian-11-r14 (#53330)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.2.13-debian-11-r6 (#53331)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/supabase-postgres] Release 15.1.0-debian-11-r189 (#53332)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql] Release 14.10.0-debian-11-r17 (#53333)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.1.26-debian-11-r5 (#53334)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql-repmgr] Release 15.5.0-debian-11-r10 (#53336)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.2.13-debian-11-r7 (#53335)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql-repmgr] Release 13.13.0-debian-11-r10 (#53337)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql-repmgr] Release 16.1.0-debian-11-r11 (#53338)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql] Release 16.1.0-debian-11-r15 (#53339)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/spark] Release 3.4.2-debian-11-r0 (#53340)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql-repmgr] Release 14.10.0-debian-11-r10 (#53341)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgresql] Release 15.5.0-debian-11-r15 (#53342)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/aws-cli] Release 2.14.5-debian-11-r0 (#53343)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/opensearch] Release 2.11.1-debian-11-r0 (#53344)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rabbitmq] Release 3.11.26-debian-11-r1 (#53346)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rabbitmq] Release 3.10.25-debian-11-r74 (#53347)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rabbitmq] Release 3.12.10-debian-11-r1 (#53348)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/jwt-cli] Release 6.0.0-debian-11-r23 (#53350)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/clickhouse] Deprecate Clickhouse branch 22.8 (#53349)

Deprecate Clickhouse 22.8

Signed-off-by: Carlos Rodríguez Hernández <carlosrh@vmware.com>

Commit:
[bitnami/neo4j] Release 5.14.0-debian-11-r0 (#53351)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/ksql] Release 7.4.3-debian-11-r0 (#53354)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/ksql] Release 7.3.6-debian-11-r0 (#53355)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/schema-registry] Release 7.3.6-debian-11-r0 (#53356)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/schema-registry] Release 7.4.3-debian-11-r0 (#53357)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

fmulero committed:
[bitnami/containers] Remove unnecessary workflows (#53315)

Signed-off-by: Fran Mulero <fmulero@vmware.com>

dcamarenaAZO committed:
[bitnami/postegresql-repmgr] Added missing postgresql environment variables (#52254)

Added postgresql environment variables regarding
connections to README.md. Defa

Signed-off-by: Daniel Camarena <daniel.camarena@azo.com>

Commit:
[bitnami/tomcat] Release 10.1.16-debian-11-r4 (#53362)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/tomcat] Release 9.0.83-debian-11-r4 (#53363)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/tomcat] Release 8.5.96-debian-11-r4 (#53364)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/tomcat] Release 10.1.16-debian-11-r5 (#53365)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

hedrickbt committed:
[bitnami/moodle] Handle --prefix (table prefix) being overridden via MOODLE_INSTALL_EXTRA_ARGS (#51278)

* Handle --prefix (table prefix) being overridden via MOODLE_INSTALL_EXTRA_ARGS

At the moment libmoodle.sh assumes a table prefix of the default mdl_ .  When someone overrides this with the --prefix=xxx, then there are errors with the sql statements in libmoodle.sh that reference tables with mdl_....

Signed-off-by: Brooke Hedrick <brooke.t.hedrick@gmail.com>

* Update libmoodle.sh

Part 2 of 2 for --prefix support.  Using the new mdl_prefix variable.

Signed-off-by: Brooke Hedrick <brooke.t.hedrick@gmail.com>

* Update bitnami/moodle/4.2/debian-11/rootfs/opt/bitnami/scripts/libmoodle.sh

Co-authored-by: Michiel <mdhont79@gmail.com>
Signed-off-by: Brooke Hedrick <brooke.t.hedrick@gmail.com>

* Update libmoodle.sh

simplified version

Signed-off-by: Brooke Hedrick <brooke.t.hedrick@gmail.com>

---------

Signed-off-by: Brooke Hedrick <brooke.t.hedrick@gmail.com>
Co-authored-by: Michiel <mdhont79@gmail.com>

Commit:
[bitnami/python] Release 3.12.0-debian-11-r10 (#53366)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/percona-xtrabackup] Release 8.0.35-30-debian-11-r0 (#53367)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/minio-client] Release 2023.12.2-debian-11-r0 (#53368)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.2.13-debian-11-r9 (#53370)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/symfony] Release 5.4.33-debian-11-r0 (#53371)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.11.6-debian-11-r15 (#53372)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.3.0-debian-11-r2 (#53373)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.8.18-debian-11-r48 (#53374)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.1.26-debian-11-r7 (#53375)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.10.13-debian-11-r47 (#53376)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/nats] Release 2.10.6-debian-11-r0 (#53377)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.9.18-debian-11-r49 (#53378)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.12.0-debian-11-r11 (#53379)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/symfony] Release 6.3.10-debian-11-r0 (#53380)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.10.13-debian-11-r48 (#53381)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/pymilvus] Release 2.3.4-debian-11-r0 (#53382)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.9.18-debian-11-r50 (#53383)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mastodon] Release 4.2.2-debian-11-r0 (#53384)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.8.18-debian-11-r49 (#53385)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/grafana-image-renderer] Release 3.9.0-debian-11-r0 (#53386)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.1.26-debian-11-r8 (#53387)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/jax] Release 0.4.21-debian-11-r0 (#53388)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/azure-cli] Release 2.55.0-debian-11-r0 (#53389)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/python] Release 3.11.7-debian-11-r0 (#53390)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/couchdb] Release 3.3.3-debian-11-r0 (#53391)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/suitecrm] Release 7.14.2-debian-11-r3 (#53393)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mongodb] Release 5.0.23-debian-11-r2 (#53394)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/suitecrm] Release 8.4.2-debian-11-r3 (#53395)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mongodb] Release 6.0.12-debian-11-r2 (#53396)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mongodb-sharded] Release 6.0.12-debian-11-r2 (#53397)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mongodb-sharded] Release 5.0.23-debian-11-r2 (#53398)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
🔄 synced file(s) with bitnami/support (#53399)

🔄 synced local '.github/workflows/' with remote 'workflows/'

Signed-off-by: bitnami-bot <bitnami-bot@vmware.com>

Commit:
[bitnami/sealed-secrets-kubeseal] Release 0.24.4-debian-11-r0 (#53400)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/aws-cli] Release 1.31.7-debian-11-r0 (#53401)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mastodon] Release 4.2.3-debian-11-r0 (#53402)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/cosign] Release 2.2.2-debian-11-r0 (#53404)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/haproxy] Release 2.9.0-debian-11-r0 (#53405)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/aws-cli] Release 1.31.8-debian-11-r0 (#53406)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/pgbouncer] Release 1.21.0-debian-11-r6 (#53408)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/wildfly] Release 30.0.1-debian-11-r0 (#53410)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.1.26-debian-11-r9 (#53411)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/fluxcd-notification-controller] Release 1.2.0-debian-11-r0 (#53412)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/airflow-exporter] Release 0.20220314.0-debian-11-r441 (#53413)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/acmesolver] Release 1.13.2-debian-11-r2 (#53414)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/blackbox-exporter] Release 0.24.0-debian-11-r141 (#53415)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/alertmanager] Release 0.26.0-debian-11-r45 (#53416)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/apisix-ingress-controller] Release 1.7.1-debian-11-r1 (#53417)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/cert-manager-webhook] Release 1.13.2-debian-11-r2 (#53418)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/cluster-autoscaler] Release 1.28.1-debian-11-r1 (#53419)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/cainjector] Release 1.13.2-debian-11-r2 (#53420)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/consul-exporter] Release 0.9.0-debian-11-r301 (#53421)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/contour] Release 1.25.3-debian-11-r3 (#53422)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/consul] Release 1.17.0-debian-11-r2 (#53423)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/cert-manager] Release 1.13.2-debian-11-r2 (#53424)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/contour] Release 1.26.1-debian-11-r4 (#53425)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/apache-exporter] Release 1.0.3-debian-11-r2 (#53426)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/configmap-reload] Release 0.12.0-debian-11-r9 (#53427)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/cosign] Release 2.2.2-debian-11-r1 (#53428)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/contour] Release 1.27.0-debian-11-r1 (#53429)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/dex] Release 2.37.0-debian-11-r100 (#53430)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/elasticsearch-exporter] Release 1.6.0-debian-11-r97 (#53431)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/fluxcd-image-automation-controller] Release 0.36.1-debian-11-r9 (#53432)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/etcd] Release 3.4.28-debian-11-r1 (#53433)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/fluxcd-notification-controller] Release 1.2.0-debian-11-r1 (#53434)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/external-dns] Release 0.14.0-debian-11-r2 (#53435)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/fluxcd-helm-controller] Release 0.36.2-debian-11-r2 (#53436)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/gitlab-runner-helper] Release 16.6.1-debian-11-r2 (#53437)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/fluxcd-image-reflector-controller] Release 0.30.0-debian-11-r45 (#53438)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/gotrue] Release 1.0.1-debian-11-r218 (#53439)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/fluxcd-kustomize-controller] Release 1.1.1-debian-11-r3 (#53440)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/grafana-mimir] Release 2.10.4-debian-11-r2 (#53441)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/grafana-operator] Release 5.5.2-debian-11-r1 (#53442)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/gitlab-runner] Release 16.6.1-debian-11-r2 (#53443)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/grafana] Release 10.2.2-debian-11-r3 (#53444)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/grafana-tempo-query] Release 2.3.1-debian-11-r1 (#53445)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/grafana-tempo-vulture] Release 2.3.1-debian-11-r1 (#53446)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/haproxy] Release 2.9.0-debian-11-r1 (#53447)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/golang] Release 1.20.12-debian-11-r0 (#53448)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/golang] Release 1.21.5-debian-11-r0 (#53449)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/argo-workflow-controller] Release 3.5.2-debian-11-r1 (#53450)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/grafana-tempo] Release 2.3.1-debian-11-r1 (#53451)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/etcd] Release 3.5.10-debian-11-r3 (#53452)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/harbor-adapter-trivy] Release 2.9.1-debian-11-r1 (#53453)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/harbor-core] Release 2.9.1-debian-11-r2 (#53454)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/grafana-loki] Release 2.9.2-debian-11-r2 (#53455)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mongodb-exporter] Release 0.40.0-debian-11-r1 (#53456)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/harbor-registryctl] Release 2.9.1-debian-11-r2 (#53457)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/promtail] Release 2.9.2-debian-11-r2 (#53458)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kaniko] Release 1.19.0-debian-11-r1 (#53459)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/oras] Release 1.1.0-debian-11-r28 (#53460)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/minio-client] Release 2023.12.2-debian-11-r1 (#53461)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/prometheus-operator] Release 0.70.0-debian-11-r1 (#53462)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kong-ingress-controller] Release 3.0.1-debian-11-r1 (#53463)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/harbor-registry] Release 2.9.1-debian-11-r2 (#53464)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/harbor-exporter] Release 2.9.1-debian-11-r2 (#53465)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kubernetes-event-exporter] Release 1.6.1-debian-11-r1 (#53466)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kubectl] Release 1.27.8-debian-11-r2 (#53467)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/harbor-jobservice] Release 2.9.1-debian-11-r3 (#53468)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/nats-exporter] Release 0.13.0-debian-11-r2 (#53469)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kube-state-metrics] Release 2.10.1-debian-11-r1 (#53470)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kubeapps-apprepository-controller] Release 2.9.0-debian-11-r13 (#53472)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kubectl] Release 1.28.4-debian-11-r2 (#53471)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kubectl] Release 1.26.11-debian-11-r2 (#53473)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rmq-default-credential-updater] Release 1.0.4-debian-11-r3 (#53474)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/pinniped] Release 0.27.0-debian-11-r2 (#53475)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/sealed-secrets] Release 0.24.4-debian-11-r1 (#53476)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/thanos] Release 0.32.5-debian-11-r2 (#53477)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rmq-messaging-topology-operator] Release 1.12.1-debian-11-r3 (#53478)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.2.13-debian-11-r10 (#53479)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/postgres-exporter] Release 0.15.0-debian-11-r3 (#53480)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/whereabouts] Release 0.6.2-debian-11-r111 (#53481)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/argo-workflow-cli] Release 3.5.2-debian-11-r1 (#53482)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/node-exporter] Release 1.7.0-debian-11-r1 (#53483)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kapacitor] Release 1.7.1-debian-11-r2 (#53484)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/jaeger] Release 1.51.0-debian-11-r2 (#53485)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rabbitmq-cluster-operator] Release 2.6.0-debian-11-r1 (#53486)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/wordpress] Release 6.4.1-debian-11-r13 (#53487)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/nats] Release 2.10.6-debian-11-r1 (#53488)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mysqld-exporter] Release 0.15.0-debian-11-r72 (#53489)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/metrics-server] Release 0.6.4-debian-11-r73 (#53490)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/metallb-controller] Release 0.13.12-debian-11-r2 (#53491)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.1.26-debian-11-r10 (#53492)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/memcached-exporter] Release 0.14.1-debian-11-r0 (#53493)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/pushgateway] Release 1.6.2-debian-11-r20 (#53494)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/multus-cni] Release 4.0.2-debian-11-r129 (#53495)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/php-fpm] Release 8.3.0-debian-11-r3 (#53496)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/vault-csi-provider] Release 1.4.1-debian-11-r3 (#53497)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/metallb-speaker] Release 0.13.12-debian-11-r2 (#53498)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/vault-k8s] Release 1.3.1-debian-11-r3 (#53499)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kapacitor] Release 1.7.1-debian-11-r3 (#53500)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rclone] Release 1.65.0-debian-11-r1 (#53501)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/telegraf] Release 1.28.5-debian-11-r1 (#53502)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/sealed-secrets-kubeseal] Release 0.24.4-debian-11-r1 (#53503)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/gitea] Release 1.21.1-debian-11-r2 (#53504)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kiam] Release 4.2.0-debian-11-r423 (#53506)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/aws-cli] Release 2.14.6-debian-11-r0 (#53507)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/apache] Release 2.4.58-debian-11-r3 (#53508)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/aws-cli] Release 1.31.9-debian-11-r0 (#53509)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kong-ingress-controller] Release 2.12.2-debian-11-r1 (#53510)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/laravel] Release 10.2.10-debian-11-r0 (#53511)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/cluster-autoscaler] Release 1.28.2-debian-11-r0 (#53512)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/keycloak] Release 22.0.5-debian-11-r4 (#53513)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/minio] Release 2023.12.6-debian-11-r0 (#53514)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/nats] Release 2.10.7-debian-11-r0 (#53515)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/spring-cloud-dataflow] Release 2.11.1-debian-11-r2 (#53516)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/tomcat] Release 9.0.83-debian-11-r5 (#53517)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/airflow-scheduler] Release 2.7.3-debian-11-r3 (#53518)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/airflow-worker] Release 2.7.3-debian-11-r4 (#53519)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rabbitmq] Release 3.10.25-debian-11-r75 (#53521)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rabbitmq] Release 3.11.26-debian-11-r2 (#53522)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/rabbitmq] Release 3.12.10-debian-11-r2 (#53523)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/airflow] Release 2.7.3-debian-11-r4 (#53524)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/apisix] Release 3.7.0-debian-11-r1 (#53525)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/aspnet-core] Release 6.0.25-debian-11-r1 (#53526)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/aspnet-core] Release 8.0.0-debian-11-r2 (#53527)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/aspnet-core] Release 7.0.14-debian-11-r1 (#53528)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/codeigniter] Release 4.4.3-debian-11-r3 (#53529)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/discourse] Release 3.1.3-debian-11-r3 (#53530)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/dotnet] Release 6.0.25-debian-11-r1 (#53532)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/dokuwiki] Release 20230404.1.0-debian-11-r105 (#53531)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/dotnet-sdk] Release 8.0.100-debian-11-r1 (#53533)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/dotnet] Release 7.0.14-debian-11-r1 (#53535)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/dotnet-sdk] Release 7.0.404-debian-11-r1 (#53534)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/dotnet-sdk] Release 6.0.417-debian-11-r1 (#53536)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/dotnet] Release 8.0.0-debian-11-r1 (#53537)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/drupal] Release 10.1.6-debian-11-r6 (#53538)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/drupal-nginx] Release 10.1.6-debian-11-r6 (#53539)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/elasticsearch] Release 7.17.15-debian-11-r1 (#53540)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/elasticsearch] Release 8.11.1-debian-11-r1 (#53541)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/express] Release 4.18.2-debian-11-r170 (#53542)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/flink] Release 1.18.0-debian-11-r2 (#53543)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/git] Release 2.43.0-debian-11-r1 (#53544)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/gradle] Release 8.5.0-debian-11-r1 (#53545)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/java] Release 1.8.392-9-debian-11-r7 (#53546)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/jasperreports] Release 8.2.0-debian-11-r98 (#53547)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/java] Release 11.0.21-10-debian-11-r6 (#53548)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/java] Release 17.0.9-11-debian-11-r7 (#53549)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/java] Release 21.0.1-12-debian-11-r4 (#53550)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/jenkins-agent] Release 0.3192.0-debian-11-r2 (#53552)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/joomla] Release 5.0.1-debian-11-r1 (#53553)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/jenkins] Release 2.426.1-debian-11-r1 (#53554)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/jupyter-base-notebook] Release 4.0.2-debian-11-r68 (#53555)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/jupyterhub] Release 4.0.2-debian-11-r67 (#53556)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kafka] Release 3.2.3-debian-11-r228 (#53557)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kafka] Release 3.3.2-debian-11-r242 (#53558)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kafka] Release 3.4.1-debian-11-r116 (#53559)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kibana] Release 7.17.15-debian-11-r1 (#53560)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kafka] Release 3.6.0-debian-11-r3 (#53561)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kafka] Release 3.5.1-debian-11-r75 (#53562)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kibana] Release 8.11.1-debian-11-r2 (#53563)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kubeapps-dashboard] Release 2.9.0-debian-11-r18 (#53564)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/laravel] Release 9.5.2-debian-11-r129 (#53565)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/matomo] Release 4.16.0-debian-11-r2 (#53567)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/magento] Release 2.4.6-debian-11-r117 (#53566)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mediawiki] Release 1.40.1-debian-11-r8 (#53568)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mlflow] Release 2.8.1-debian-11-r1 (#53569)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kubeapps-asset-syncer] Release 2.9.0-debian-11-r14 (#53570)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/ksql] Release 7.1.10-debian-11-r1 (#53571)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kube-rbac-proxy] Release 0.15.0-debian-11-r2 (#53572)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kubeapps-apis] Release 2.9.0-debian-11-r14 (#53573)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mongodb-sharded] Release 7.0.4-debian-11-r1 (#53574)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/kafka-exporter] Release 1.7.0-debian-11-r134 (#53575)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/mongodb] Release 7.0.4-debian-11-r1 (#53576)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/moodle] Release 4.1.6-debian-11-r6 (#53577)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/moodle] Release 4.2.3-debian-11-r4 (#53578)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/moodle] Release 4.3.0-debian-11-r4 (#53579)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/natscli] Release 0.1.1-debian-11-r2 (#53580)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/nginx-exporter] Release 0.11.0-debian-11-r371 (#53581)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/nginx-ingress-controller] Release 1.9.4-debian-11-r3 (#53582)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/node] Release 16.20.2-debian-11-r53 (#53583)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/node] Release 18.19.0-debian-11-r1 (#53584)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

Commit:
[bitnami/node] Release 20.10.0-debian-11-r1 (#53585)

Signed-off-by: Bitnami Containers <bitnami-bot@vmware.com>

christian-scheible committed:
Removed comments, fixed indentation.

christian-scheible commented on bitnami/tomcat/10.1/debian-11/rootfs/opt/bitnami/scripts/libtomcat.sh:
```diff
@@ -163,11 +163,16 @@ tomcat_enable_application() {
 tomcat_initialize() {
     if ! is_empty_value "$TOMCAT_EXTRA_JAVA_OPTS"; then
-        cat >>"${TOMCAT_BIN_DIR}/setenv.sh" <<EOF
+      # Check if $TOMCAT_EXTRA_JAVA_OPTS has already been added to setenv.sh
```
Hi it seems that I have now broken it by rebasing the branch. Should I start a new branch an a new pull request?

andresbono replied:
Hi, that is fine, you can open a new PR if you prefer in order to restore the changes.

christian-scheible replied:
Ok I made a new one: https://github.com/bitnami/containers/pull/53780
